Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -113,7 +116,17 @@ vtkMRMLSliceNode::vtkMRMLSliceNode()
   this->LayoutColor[1] = vtkMRMLSliceNode::grayColor()[1];
   this->LayoutColor[2] = vtkMRMLSliceNode::grayColor()[2];
 
-  this->SetOrientationToAxial();
+  this->SliceToRAS->SetElement(0, 0, -1.0);
+  this->SliceToRAS->SetElement(1, 0,  0.0);
+  this->SliceToRAS->SetElement(2, 0,  0.0);
+  this->SliceToRAS->SetElement(0, 1,  0.0);
+  this->SliceToRAS->SetElement(1, 1,  1.0);
+  this->SliceToRAS->SetElement(2, 1,  0.0);
+  this->SliceToRAS->SetElement(0, 2,  0.0);
+  this->SliceToRAS->SetElement(1, 2,  0.0);
+  this->SliceToRAS->SetElement(2, 2,  1.0);
```

Comment:
Yes. 

The issue is that having the Axial matrix set to in two places is not ideal. That is why setting to Identity here and relaying on the presets make more sense